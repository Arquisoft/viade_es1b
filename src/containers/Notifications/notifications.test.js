import React from 'react';
import { cleanup, render, queryByTestId, act } from '@testing-library/react';
import NotificationList from './NotificationsList';

let wrapper;
beforeEach(() => act(() => {
  const { container, debug } = render(
    <NotificationList />
  );
  wrapper = container;
  //debug();
}
));

describe('Login', () => {
  afterAll(cleanup);
  
  test('renders without crashing', () => {
    expect(wrapper).toBeTruthy();
  });

  test('renders title properly', () => {
    expect(queryByTestId(wrapper, "notifications-title")).not.toBeNull();
  });

  test('renders notifications list ', () => {
    expect(queryByTestId(wrapper, "notifications-list")).not.toBeNull();
    
  });
});