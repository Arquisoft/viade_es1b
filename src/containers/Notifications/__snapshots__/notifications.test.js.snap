// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders with default props 1`] = `ShallowWrapper {}`;
